WinUXTheme.m

```diff
@@ -58,8 +58,11 @@ + (void) initialize
   [NSImageRep registerImageRepClass:[WIN32VSImageRep class]];
 
   // See WinNSTabView.m for implemented usage...
-  [[NSUserDefaults standardUserDefaults] registerDefaults:@{@"GSMinimumTabHeight" : @12.0 }];
-  [[NSUserDefaults standardUserDefaults] registerDefaults:@{@"GSMaximumTabHeightPrivate" : @31.0 }];
+  [[NSUserDefaults standardUserDefaults] registerDefaults:
+    [NSDictionary dictionaryWithObjectsAndKeys:
+      @12.0, @"GSMinimumTabHeight",
```

Comment:
It is even worse, you will also need to replace the short NSNumber notation. Sorry.